CHANGELOG.md

```diff
@@ -2,7 +2,27 @@
 
 This file documents all notable changes to the HEMCO-CESM interface since late 2022.
 
-## [1.4.0] - TBD
+## [2.2] - TBD
+### Changed
+- Added new default configuration file for HEMCO which reads/regrids 3D emissions hourly
+- Updated debug printout and message for where to find error messages
+
+### Fixed
+- Fixed HEMCO clock to be time at start of timestep rather than end
+- Fixed formatting for Ap/Bp printout
+- Fixed condition for first timestep to ensure HEMCO is not run at timestep 0
+
+## [2.1] - 2024-07-31
```

Comment:
Should semantic versioning be used and always keep x.y.z, or did you intend to switch to just x.y versioning?

Reply:
Yes, I will update that to be 2.2.0. Regarding 2.1, there is a tag for 2.1.1 but not 2.1.0. Do you remember what happened with 2.0.0?

Reply:
There is also no tag for 1.4. Let me know if there are any changes I should make to the changelog for these older versions. I tried to fill it in as best I could starting from where it was before.

Reply:
I think that 2.1.1 tag `hemco_cesm2_1_1_gc_rel_01` is misleadingly named, it was probably for IGC10 and 2.1.1 refers to the CESM version, not HEMCO-CESM version. It's from 2 years ago. So there was never a 2.1 tag.

1.4.0 was in development and I think everything was rolled into 2.0.0. It's probably a good idea to clean up the branches sometime.